xapian-bindings/android/android.i

```diff
@@ -0,0 +1,401 @@
+%module(directors="1") xapian
```

Comment:
My thought would be to just conditionalise any required differences between "normal" Java and Android in the existing Java bindings.  Otherwise we'll have to make sure any change to Java is also applied to the Android bindings (or isn't appropriate) which is a lot of fiddly patch tracking.